src/RpcServer/RpcServer.Node.cs

```diff
@@ -60,7 +60,7 @@ private static JObject GetRelayResult(VerifyResult reason, UInt256 hash)
             }
             else
             {
-                throw new RpcException(-500, reason.ToString());
+                throw new RpcException(RpcErrorFactory.VerificationFailed(reason.ToString()));
```

Comment:
This comment is still relevant, we need to carefully distinguish between various relay failure reasons, because it's important for users applications. All these commented errors are for `GetRelayResult`:
```
        public static readonly RpcError AlreadyExists = new(-504, "Inventory already exists");
        //public static readonly RpcError MempoolCapReached = new(-505, "Memory pool capacity reached");
        //public static readonly RpcError AlreadyInPool = new(-506, "Already in pool");
        //public static readonly RpcError InsufficientNetworkFee = new(-507, "Insufficient network fee");
        //public static readonly RpcError PolicyFailed = new(-508, "Policy check failed");
        //public static readonly RpcError InvalidScript = new(-509, "Invalid transaction script");
        //public static readonly RpcError InvalidAttribute = new(-510, "Invalid transaction attribute");
        //public static readonly RpcError InvalidSize = new(-511, "Invalid inventory size");
        //public static readonly RpcError ExpiredTransaction = new(-512, "Expired transaction");
```